src/main/java/io/cryostat/net/web/http/api/v1/TargetRecordingDeleteHandler.java

```diff
@@ -102,7 +105,12 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
         String recordingName = ctx.pathParam("recordingName");
         ConnectionDescriptor connectionDescriptor = getConnectionDescriptorFromContext(ctx);
         try {
-            recordingTargetHelper.deleteRecording(connectionDescriptor, recordingName).get();
+            Matcher m = SNAPSHOT_NAME_PATTERN.matcher(recordingName);
+            if (m.matches()) {
+                recordingTargetHelper.deleteSnapshot(connectionDescriptor, recordingName).get();
+            } else {
+                recordingTargetHelper.deleteRecording(connectionDescriptor, recordingName).get();
+            }
```

Comment:
This looks useful enough to extract out into a utility method within the `RecordingTargetHelper`, and that also helps encapsulate and consolidate the logic there so that this API handler is as simple as possible. ie the handler doesn't know anything about snapshot or not, it just knows how to delete active recordings by name, and the Helper is what figures out how to handle that exactly.

Reply:
I extracted determining whether a recording is a snapshot or not into the `RecordingTargetHelper` but I also got rid of the separate `deleteSnapshot` and `deleteRecording` methods. I feel it's less cluttered this way with the only recording deletion methods being `deleteRecording(ConnectionDescriptor connectionDescriptor, String recordingName` and `deleteRecording(ConnectionDescriptor connectionDescriptor, String recordingName, boolean isSnapshot, boolean issueNotification)`.